README.md

```diff
@@ -13,35 +13,47 @@ Keep calm and add Geocoder to your `mix.exs` dependencies:
 
 ```elixir
 def deps do
-  [{:geocoder, "~> 0.7"}]
+  [{:geocoder, "~> 1.0"}]
 end
 ```
 
-Add it to your starting applications too:
+Update your mix dependencies:
+
+```
+mix deps.get
+```
+
+Add :geocoder to your applications list if your Elixir version is 1.3 or lower:
```

Comment:
Your PR removes Elixir versions < 1.6 from the CI build matrix (fine by me, couldn't get the lib to build on Elixir 1.4 @ Erlang/OTP 19.3).

If we decide to go that route (i.e. dropping older Elixir versions), we should also change the `elixir: "1.2` in `mix.exs` to 1.6 and remove this README hint for versions that aren't supported anymore anyway.